doc/development_process/dev_env_and_tools.rst

```diff
@@ -190,8 +190,10 @@ Closing Stale Issues and Pull Requests
   Use the mailing lists for discussions.
 - In case of both issues and pull-requests the original poster needs to respond
   to questions and provide clarifications regarding the issue or the change.
-  Failing to do so, an issue or a pull request will be closed automatically
-  after 6 months.
+  After one week without a response to a request, a second attempt to elicit
+  a response from the contributor will be made. After one more week without a
+  response the item may be closed at the discretion of the appropriate working
```

Comment:
> at the discretion of the appropriate working group

Bike-shedding a bit: most work is done by maintainers or their own, not workgroups. So I'd say it would be better to say "at the discretion of the involved maintainers" which is anyway generic enough (I really hope you did not mean to propose we have a meeting each time we need to close a PR or issue :) )

Reply:
We already have a meeting that reviews all the PR and bugs and we propose changes on these items. On Tuesdays. 

But yes, I was intending this to be the working group to do this and not necessarily an individual maintainer. The thinking here was to have a tool/filter that could identify these stale PR/Issues and have them marked stale or ready to close. The meeting could quickly go through them and make decision on close or not close. Note that this only occurs if there is a specific request for additional information and the contributor is not responding.

Reply:
> We already have a meeting that reviews all the PR and bugs and we propose changes on these items. On Tuesdays.

How things are nowadays, the "bug triage and release readiness meeting" (I presume that is the one you refer to) does certainly not cover all open PRs and issues.
Only a very small part of the open issues, and only the ones that are not yet fully triaged, are glimpsed into, and only for the purpose of quickly assigning them to somebody and stamping a priority on them. In that meeting, they are certainly not "reviewed".
Apart from this, this meeting is not a working group forum. It is just the release manager and "whoever shows up" for that meeting. And in general that group is not, at all, the same set of people which is reviewing the open set of PRs.

> I was intending this to be the working group to do this and not necessarily an individual maintainer. 

So what you are proposing is to change the scope of that meeting to be "bug triage, release readiness AND closing of stale PRs and issues"..
Ok.. I guess you could try to..
(defining this here seems quite far from the action you got to note that after 10 days without response issues are closed)

Anyhow, again, how this paragraph is phrased*, it gives the impression that, in general, PRs and issues are reviewed by "working groups" which is not at all how things are, or could be in the foreseeable future.

*Together with the line "Zephyr community members work together with project working groups
to review these Issues and PRs"

Reply:
If you just replace "After one more week without a response the item may be closed at the discretion of the appropriate working group reviewing the item (draft and DNM tagged pull requests are excluded)." 
with 
"After one more week without a response the item may be closed (draft and DNM tagged pull requests are excluded)." 
or with 
"After one more week without a response the item may be closed by the relevant maintainers (draft and DNM tagged pull requests are excluded)." 
I'd be ok with it.
("Relevant maintainers" is generic and vague enough, it can be a set of people in a working group, whoever was in a "closing stale issues and PRs" meeting, or the area maintainer(s) )